--- test_imgwarp.cpp.orig	2026-08-29 03:07:02
+++ test_imgwarp.cpp	2026-09-03 03:58:36
@@ -39,8 +39,6 @@
 //
 //M*/
 
-#include "opencv2/ts/ocl_test.hpp"
-#include "opencv2/ts/ts_gtest.h"
 #include "test_precomp.hpp"
 
 namespace opencv_test { namespace {
@@ -1121,82 +1119,6 @@
     float ca = cosf(angle), sa = sinf(angle);
     M[0] = scale*ca; M[1] = scale*sa; M[2] = scale*(-cx*ca - cy*sa) + cx;
     M[3] = -scale*sa; M[4] = scale*ca; M[5] = scale*(cx*sa - cy*ca) + cy;
-}
-
-TEST(Imgproc_Warping, DISABLED_playground)
-{
-    int imgtype = CV_32F;
-    int imgcn = 3;
-    bool useOpenCL = true;
-
-    auto ts = cvtest::TS::ptr();
-    Mat img0 = imread(string(ts->get_data_path()) + "stereomatching/datasets/tsukuba/im2.png", 1), img1, img;
-    int iangle = -1;
-    int borderType = BORDER_CONSTANT;
-    Scalar borderValue(0, 128, 0);
-
-    double cvtscale = imgtype == CV_16U ? 256. : imgtype == CV_32F ? 1./255 : 1.;
-    if (imgcn == 1) {
-        cvtColor(img0, img1, COLOR_BGR2GRAY);
-    } else if (imgcn == 4) {
-        cvtColor(img0, img1, COLOR_BGR2BGRA);
-    } else if (imgcn == 3) {
-        img1 = img0;
-    } else {
-        CV_Assert(imgcn == 2);
-        std::vector<Mat> ch;
-        split(img0, ch);
-        ch.pop_back();
-        merge(ch, img1);
-    }
-    img1.convertTo(img, imgtype, cvtscale);
-    Mat canvas0(img.size(), imgtype), canvas8;
-    float cx = img.cols*0.5f, cy = img.rows*0.5f;
-    if (img.depth() == CV_32F) {
-        borderValue = Scalar(100*cvtscale, 0*cvtscale, 100*cvtscale);
-    }
-    UMat uimg, ucanvas;
-    if (useOpenCL) {
-        img.copyTo(uimg);
-    }
-
-    for(;;) {
-        iangle = (iangle + 1) % (360*4);
-        float angle = float(iangle*CV_PI/180.f*0.25f);
-        float scale = float(1 + 0.2f*sin(angle));
-        float Mdata[9] = {1, 0, 0, 0, 1, 0, 0, 0, 1};
-        rotation2affine(scale, angle, cx, cy, Mdata);
-        Mat M(2, 3, CV_32F, Mdata);
-
-        double t0 = (double)getTickCount();
-        if (!useOpenCL) {
-            warpAffine(img, canvas0, M, canvas0.size(), INTER_CUBIC, borderType, borderValue);
-        } else {
-            warpAffine(uimg, ucanvas, M, uimg.size(), INTER_CUBIC, borderType, borderValue);
-            ocl::finish();
-        }
-        t0 = (double)getTickCount() - t0;
-
-        if (useOpenCL) {
-            ucanvas.copyTo(canvas0);
-        }
-        canvas0.convertTo(canvas8, CV_8U, 1./cvtscale);
-        if (canvas8.channels() == 2) {
-            std::vector<Mat> ch;
-            split(canvas8, ch);
-            ch.push_back(Mat::zeros(canvas8.size(), CV_8U));
-            merge(ch, canvas8);
-        }
-        printf("opencv time = %.1fms\n", t0*1000./getTickFrequency());
-        imshow("result (opencv)", canvas8);
-        int c = waitKey(1);
-        if (c < 0)
-            continue;
-        if ((c & 255) == 27)
-            break;
-        if ((waitKey() & 255) == 27)
-            break;
-    }
 }
 
 }} // namespace
