--- ts.hpp.orig	2026-08-29 03:07:11
+++ ts.hpp	2026-09-02 23:37:13
@@ -9,9 +9,6 @@
 
 #include "opencv2/core.hpp"
 #include "opencv2/imgproc.hpp"
-#include "opencv2/imgcodecs.hpp"
-#include "opencv2/videoio.hpp"
-#include "opencv2/highgui.hpp"
 
 #include "opencv2/core/utility.hpp"
 
@@ -125,7 +122,7 @@
 #pragma clang diagnostic ignored "-Wdeprecated-copy"
 #pragma clang diagnostic ignored "-Winconsistent-missing-override"
 #endif
-#include "opencv2/ts/ts_gtest.h"
+#include <gtest/gtest.h>
 #if defined(__OPENCV_BUILD) && defined(__GNUC__) && __GNUC__ >= 5
 //#pragma GCC diagnostic pop
 #endif
